README.md

```diff
@@ -2,197 +2,68 @@
-
-Once you have the necessary prerequisites, LiT can be compiled by running the following
-commands:
+## Execution
+Run Lightning Terminal with a local LND instance:
 
 ```
-git clone https://github.com/lightninglabs/lightning-terminal.git
-cd lightning-terminal
-make install
+./litd --uipassword=UP48lm4VjqxmOxB9X9stry6VTKBRQI
 ```
 
-This will produce the `litd` executable and add it to your `GOPATH`. The CLI binaries for
-`lncli`, `loop`, and `frcli` are not created by `make install`. You will need to download
-those binaries from the [lnd](https://github.com/lightningnetwork/lnd/releases),
-[loop](https://github.com/lightninglabs/loop/releases), and
-[faraday](https://github.com/lightninglabs/faraday/releases) repos manually.
-
-#### Executing CLI Commands
+Visit https://localhost:8443 to access Terminal.
 
-When executing `loop` and `frcli` commands, you will need to specify the connection info
-since the daemons are now integrated into `lnd`'s GRPC server.
+Note that a password with a minimum of 8 characters is required to run Lightning Terminal. In a production environment, it's recommended that you store this password as an environment variable.
 
-Examples:
+To use LiT with a remote LND instance please [follow these instructions](./doc/config-lnd-remote.md). If you would like to replace your existing LND instance with the one integrated within LiT please see [configuring Terminal](./doc/config-lnd-integrated.md).
 
-```
-loop --rpcserver=localhost:10009 --tlscertpath=$HOME/.lnd/tls.cert --macaroonpath=$HOME/.lnd/data/chain/bitcoin/mainnet/admin.macaroon
-```
-
-```
-frcli --rpcserver=localhost:10009 --tlscertpath=$HOME/.lnd/tls.cert --macaroonpath=$HOME/.lnd/data/chain/bitcoin/mainnet/admin.macaroon
-```
+## Interaction
+If you plan to run LiT on a remote machine but access the web-interface from your computer you may not want to deal with self-signed certificate browser warnings. To avoid these warnings configure the HTTP server to use a certificate from [Let's Encrypt](https://letsencrypt.org/). View the
+[Let's Encrypt Configuration](./doc/letsencrypt.md) doc for instructions on how to configure this.
 
-## Configuration
+## Upgrading
+If you used command line arguments with previous versions then you don't need to change anything when upgrading. 
 
-LiT only has a few configuration parameters itself.
+To upgrade from v0.1.1-alpha or earlier simply create a `lit.conf` file in your LiT directory. The default location LiT uses depends on your operating system:
+- MacOS: `~/Library/Application Support/Lit/lit.conf`
+- Linux: `~/.lit/lit.conf`
+- Windows: `~/AppData/Roaming/Lit/lit.conf`
 
-#### Required
+Move all the configuration settings specific to LiT from `lnd.conf` to `lit.conf` and remove the `lnd.` prefix from the configuration settings in `lnd.conf`. Note that any section headers (`[ Example ]`) in `lit.conf` should be removed or changed to comments (`# Example`). 
```

Comment:
I think the part about removing the `lnd.` prefix in the `lnd.conf` will probably confuse most users. Maybe this doesn't need to be mentioned at all?

Reply:
Good point. Changed up wording a bit